### PR TITLE
bug: also upload files and folders beginning with a dot

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes https://github.com/google-github-actions/upload-cloud-storage/issues/102

### CODE REVIEW

File: src/upload-helper.ts [hunk 1]
```diff
@@ -145,3 +145,34 @@ export class UploadHelper {
...
+ * match-all pattern is used instead.
+ * @return Sorted list of files in posix form.
+ */
+export async function expandGlob(
+  directoryPath: string,
+  glob: string,
+): Promise<string[]> {
+  const pth = toPosixPath(path.posix.join(directoryPath, glob));
+  const filesList = await globby([pth], {
+    dot: true,
```
Comment (bharathkkb) [inline src/upload-helper.ts:164]: Should we make this an option defaulting to false to maintain backwards compat?

Comment (sethvargo) [reply src/upload-helper.ts:164]: I find it surprising that we don't match what gsutil and gcloud do here, so I'm thinking of this as a "bug" more than a feature.

Comment (bharathkkb) [reply src/upload-helper.ts:164]: I wasnt aware thats default gsutil behavior so this sgtm. Maybe in the changelog we can include how to keep existing behavior via ignorefile?

**Review by bharathkkb — APPROVED** [review]

**Comment by landsman** [comment]
Thank you! ❤️ 🚀

**Pull request merged**